Frameworks/AddressBook/ABRecord.mm

```diff
@@ -47,12 +83,117 @@ bool ABRecordSetValue(ABRecordRef record, ABPropertyID property, CFTypeRef value
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes Properties that return a Multi-Value are not yet supported, and a few properties
+        (alternate birthday, creation/modification dates, person kind, phonetic middle name)
+        don't have direct equivalents.
 */
 CFTypeRef ABRecordCopyValue(ABRecordRef record, ABPropertyID property) {
```

Comment:
Guh, that's frustrating... it's such a long else-if too :(

---
In reply to: [74812542](https://github.com/Microsoft/WinObjC/pull/881#discussion_r74812542) [](ancestors = 74812542,74811019)